http/http.go

```diff
@@ -29,12 +29,15 @@ type Server struct {
 }
 
 type Response struct {
-	IP         net.IP `json:"ip"`
-	IPDecimal  uint64 `json:"ip_decimal"`
-	Country    string `json:"country,omitempty"`
-	CountryISO string `json:"country_iso,omitempty"`
-	City       string `json:"city,omitempty"`
-	Hostname   string `json:"hostname,omitempty"`
+	IP                           net.IP `json:"ip"`
+	IPDecimal                    uint64 `json:"ip_decimal"`
+	Country                      string `json:"country,omitempty"`
+	CountryISO                   string `json:"country_iso,omitempty"`
+	IsInEuropeanUnion            bool `json:"is_in_european_union,omitempty"`
```

Comment:
I don't really see the point of this field when latitude / longitude are available. Please remove it.

Reply:
Which field are you referring to? If it's 'IsInEuropeanUnion' I would think that is very much needed, because of GDPR. If missing, you would need to look it up from another source - which could lead to bad data.